gensrc/script/doris_builtins_functions.py

```diff
@@ -2292,7 +2294,8 @@
     'coalesce',
     'array',
     'json_array',
-    'json_object'
+    'json_object',
+    'json_search',
```

Comment:
json_search returns ALWAYS_NULLABLE, not need to change 'non_null_result' vector.